<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wndEditFirefighter</class>
 <widget class="QMainWindow" name="wndEditFirefighter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>322</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>322</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>322</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Personal Information</string>
     </attribute>
     <widget class="QComboBox" name="txtState">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>130</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>32</number>
      </property>
      <item>
       <property name="text">
        <string>AL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AK</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AZ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CO</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>HI</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>IL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>IN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>IA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>KS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>KY</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ME</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MI</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MO</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NH</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NJ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NY</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ND</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OH</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OK</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RI</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TX</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>UT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>VT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>VA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WI</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WY</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="txtStatus">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>180</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateJoin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2010</year>
        <month>8</month>
        <day>20</day>
       </datetime>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Address:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Join Date:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtCphone">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date of Birth:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtAddress">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cell Phone:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtFirstName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtMiddleName">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>210</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CDL:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>110</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Zip Code:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>160</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Work Phone:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtCDL">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>230</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtLocalID">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtHphone">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>180</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Last Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Driver's License:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>First Name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtZipCode">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>130</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>110</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>State:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtCity">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>State ID:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtStateID">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>80</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtWphone">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>180</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtLastName">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Home Phone:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Middle Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>City:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Local ID:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateDob">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2010</year>
        <month>8</month>
        <day>20</day>
       </datetime>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDrvLic">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>230</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Training</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Equipment</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Reports</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
